src/api/app/assets/javascripts/webui/application/kiwi_editor.js

```diff
@@ -259,7 +259,12 @@ $(document).ready(function(){
 
   // Revert image
   $('#kiwi-image-update-form-revert').click(function(){
-    location.reload();
+    if ($(this).hasClass('enabled')) {
+      if (confirm('Attention! All unsaved data getting lost! Continue?')) {
```

Comment:
[Nitpick] Would re-word is to:
`All unsaved data will be lost lost! Do you want to continue?'`

Reply:
>  [Nitpick] Would re-word is to: All unsaved data will be lost lost! Do you want to continue?'

Done. Also changed the message in `src/api/app/views/webui/patchinfo/_form.html.haml`